### ISSUE TITLE
[🕹️][1500 Points]

### ISSUE THREAD

**Issue by jobenjada** [formbricks/formbricks#3198]
### What side quest or challenge are you solving?

How to add a feedback box to your Vue.js app (and get notified on Slack)

### Points

1500

## This gives A LOT of points because writing a high quality technical article with good instructions, screenshots, etc. can take a full day or more 🤓

Here is the [Formbricks Writing Guide](https://formbricks.notion.site/FormTribe-Content-Writing-Guide-f6464ee837364785afdc6c88ef8bfd51?pvs=74)

### Provide proof that you've completed the task

I'll share a link to apply **with your previous work** for this task.

**Comment by Mujhtech** [comment]
Hi @jobenjada, the issue is seems to be assigned to you. Can you assign it to me?

**Comment by jobenjada** [comment]
@Mujhtech it's yours :) have fun!

**Comment by Mujhtech** [comment]
Hi @jobenjada 

Can you check this out: https://dev.to/mujhtech/how-to-add-a-feedback-box-to-your-vuejs-app-and-get-notified-on-slack-2ehc-temp-slug-9578864?preview=21eba1b7bfc5c0addea091a3cda0f6f677391edd151c39c0c6c5fcb0e270e199b3f4f13481bcff761ec134349b5ac076bcdfeae58293fcc4ed701630

**Comment by jobenjada** [comment]
@Mujhtech great work! Please release it :) if you prefer, we can also release it as guest post on formbricks.com/blog 

what do you think?

/award 750

**Comment by Mujhtech** [comment]
Sure

> @Mujhtech great work! Please release it :) if you prefer, we can also release it as guest post on formbricks.com/blog
> 
> what do you think?
> 
> /award 750

**Comment by Mujhtech** [comment]
You can share how to make a PR for that

**Comment by Mujhtech** [comment]
@jobenjada Also, it seems the point was awarded to you; can you confirm that?

**Comment by jobenjada** [comment]
@Mujhtech you are right! Please open an issue with your submission and a link to the article so I can assign the points to you :)

sorry about that, it's a bit hectic these days :D
